lighthouse-core/gather/gatherers/inspector-issues.js

```diff
@@ -45,19 +45,19 @@ class InspectorIssues extends Gatherer {
 
     driver.off('Audits.issueAdded', this._onIssueAdded);
     await driver.sendCommand('Audits.disable');
-    /** @type {Array<LH.Crdp.Audits.MixedContentIssueDetails>} */
-    const mixedContentIssues = [];
-    this._issues
-      .filter(issue => issue.code === 'MixedContentIssue')
-      .forEach(issue => {
-        if (issue.details.mixedContentIssueDetails) {
-          mixedContentIssues.push(issue.details.mixedContentIssueDetails);
-        }
-      });
-
-    return {
-      mixedContentIssues,
+    const artifact = {
+      /** @type {Array<LH.Crdp.Audits.MixedContentIssueDetails>} */
+      MixedContent: [],
     };
+
+    for (const issue of this._issues) {
+      if (issue.code === 'MixedContentIssue'
```

Comment:
Maybe drop the code check?

I wish the generated types were better here. https://github.com/ChromeDevTools/devtools-protocol/blob/master/types/protocol.d.ts#L2846